libraries/chain/transaction_context.cpp

```diff
@@ -511,10 +509,8 @@ namespace eosio { namespace chain {
       // Calculate the new highest network usage and CPU time that all of the billed accounts can afford to be billed
       auto& rl = control.get_mutable_resource_limits_manager();
       const static int64_t large_number_no_overflow = std::numeric_limits<int64_t>::max()/2;
-      int64_t account_net_limit = large_number_no_overflow;
-      int64_t account_cpu_limit = large_number_no_overflow;
-      bool greylisted_net = false;
-      bool greylisted_cpu = false;
+      int64_t account_net_limit = large_number_no_overflow,account_cpu_limit = large_number_no_overflow;
+      bool greylisted_net = false,greylisted_cpu = false;
```

Comment:
This does nothing except make it less readable.

Reply:
For which reasons? m just trying to contribute only what is correct, doesn't deteriorate and if spotted to enhance. It is also a matter of style and saving code storage space as I have explained already before. It's up to you to accept the amendment(restyling or similar in this specific case to save code storage rows) or leave it as per your view. Let us agree on this.

Reply:
What do you mean by "saving code storage space"?